scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/Meta.scala

```diff
@@ -0,0 +1,82 @@
+package scala.meta
+package internal
+package semantic
+
+import java.nio.charset.Charset
+import org.scalameta.data._
+import scala.{Seq => _}
+import scala.collection.immutable.Seq
+import scala.meta.io._
+import scala.meta.internal.semantic.{schema => s}
+import scala.meta.{semantic => m}
```

Comment:
It is. But in this project, we don't yet have an umbrella `package object meta extends Everything`.